resources/authgear/templates/en/web/authflowv2/__divider.html

```diff
@@ -0,0 +1,14 @@
+{{ define "authflowv2/__divider.html" }}
+
+<div class="divider__container {{$.Classname}}">
```

Comment:
Lets use [hr](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/hr) as the  container element.

Reply:
@Ngkaokis I think `<hr>` should be used as the outer most element, which contains the whole divider.
It is because the all elements inside `divider__container` is actually used as one horizontal rule.
And the elements inside `hr` can use non-semantic elements like div or span, because the outermost `<hr>` already tell us they are part of the horizontal rule.

Reply:
Oh, I misunderstood your meaning. Let me update it.

Reply:
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/hr

hr should not have children.